src/zcl_aff_abap_doc_reader.clas.locals_def.abap

```diff
@@ -0,0 +1,16 @@
+*"* use this source file for any type of declarations (class
+*"* definitions, interfaces or type declarations) you need for
+*"* components in the private section
+
+TYPES:
+  BEGIN OF ty_comment_block,
+    tab_comments                 TYPE string_table,
+    column_first_comment         TYPE i,
+    hook_relevant_tok_type       TYPE stokesx,
+    hook_relevant_tok_name       TYPE stokesx,
+    hook_relevant_tok_name_add   TYPE stokesx,
+    hook_relevant_tok_type_stmnt TYPE stokesx,
+    hook_relevant_tok_name_stmnt TYPE stokesx,
+  END OF ty_comment_block.
+TYPES:
+  ty_comment_blocks TYPE STANDARD TABLE OF ty_comment_block WITH EMPTY KEY.
```

Comment:
Shouldn't we move these types just to the private section of the global class? As far as I can see it is used only by the global class.

Reply:
used in https://github.com/SAP/abap-file-formats-tools/blob/hvam/cache0809/src/zcl_aff_abap_doc_reader.clas.locals_imp.abap#L24